cobc/ppparse.y

```diff
@@ -721,13 +724,53 @@ ppparse_clear_vars (const struct cb_define_struct *p)
 
 %%
 
+program_structure:
+  program_with_control_division
```

Comment:
can you move the `control_division_prefix` here before `` program_with_control_division`?